codec-http/src/main/java/io/netty/handler/codec/http/cors/CorsHandler.java

```diff
@@ -80,7 +82,11 @@ private void handlePreflight(final ChannelHandlerContext ctx, final HttpRequest
             setPreflightHeaders(response);
         }
         release(request);
-        ctx.writeAndFlush(response).addListener(ChannelFutureListener.CLOSE);
+
+        final ChannelFuture future = ctx.writeAndFlush(response);
```

Comment:
good point, I've moved both places in the CorsHandler that respond into a separate method that does the keep alive check and also sets the keep alive on the response.  There's no harm in always setting it right?  I can omit it in the case where it's 1.0 and not specified in the request if necessary.

@normanmaurer just pinging you to make sure you see this on your next review